internal/nomad/nomad_test.go

```diff
@@ -534,7 +536,10 @@ func TestAPIClient_WatchAllocationsReturnsErrorWhenAllocationCannotBeRetrievedWi
 		Type:  structs.TypeAllocationUpdated,
 		Topic: nomadApi.TopicAllocation,
 		// This should fail decoding, as Allocation.ID is expected to be a string, not int
-		Payload: map[string]interface{}{"Allocation": map[string]interface{}{"ID": 1}},
+		Payload: map[string]interface{}{"Allocation": map[string]interface{}{
+			"ID":         1,
+			"TaskStates": map[string]interface{}{TaskName: map[string]interface{}{"Restarts": 0}},
```

Comment:
Is this information on the restarts here and in line 639 still needed, or a left-over?